handbook/product-design/product-design.rituals.yml

```diff
@@ -8,3 +8,91 @@
   autoIssue:   # Enables automation of GitHub issues
     labels: [ "#g-endpoint-ops" ] # label to be applied to issue
     repo: "confidential"
+-
+  task: "Design sprint review"
+  startedOn: "2024-03-07" 
+  frequency: "Triweekly" 
+  description: "After the last estimation session, the Head of Product Design reviews the board with each group designer and de-prioritizes all design issues that were not estimated. The Head of Product Design also collects all items that are product-driven and puts them in the 🎁🗣 Feature fest meeting agenda to consider for continuing work. The number of de-prioritized issues should be recorded in the KPI spreadsheet." 
+  moreInfoUrl:  
+  dri: "noahtalerman" 
+-
+  task: "🎁 Feature fest"
+  startedOn: "2024-03-07" 
+  frequency: "Triweekly" 
+  description: "We make a decision regarding which customer and community feature requests can be committed to in the next six weeks. We create issues for any requests that don't already have one." 
+  moreInfoUrl: 
+  dri: "noahtalerman" 
+-
+  task: "🎁 Feature fest cleanup"
+  startedOn: "2024-03-07" 
+  frequency: "Triweekly" 
+  description: "Clean up the agenda in anticipation of the next meeting." 
+  moreInfoUrl: 
+  dri: "noahtalerman" 
+-
+  task: "Design sprint kickoff"
+  startedOn: "2024-03-07" 
+  frequency: "Triweekly" 
+  description: "Clean up the agenda in anticipation of the next meeting." 
+  moreInfoUrl: 
+  dri: "noahtalerman"
+-
+  task: "Sprint kickoff review"
+  startedOn: "2024-03-07" 
+  frequency: "Triweekly" 
+  description: "After each sprint kickoff, the Head of Product Design reviews the Estimated column with each group EM and de-prioritizes the features that were not included in the sprint and prepares recommended highlights for the release notes. The number of de-prioritized issues should be recorded in the KPI spreadsheet." 
+  moreInfoUrl: 
+  dri: "noahtalerman"
+-
+  task: "🦢🗣 Design review (#g-mdm)"
+  startedOn: "2024-03-07" 
+  frequency: "Daily" 
+  description: "Review designs from the MDM team." 
+  moreInfoUrl: 
+  dri: "marko-lisica"
+-
+  task: "🦢🗣 Design review (#g-endpoint-ops)"
+  startedOn: "2024-03-07" 
+  frequency: "Daily" 
+  description: "Review designs from the Endpoint ops team." 
+  moreInfoUrl: 
+  dri: "rachaelshaw"
+-
+  task: "Maintenance"
+  startedOn: "2024-03-01" 
+  frequency: "Weekly" 
+  description: "Head of Product Design checks the latest versions of relevant platforms, updates the maintenance tracker, and notifies the #help-product-design Slack channel." 
+  moreInfoUrl: 
+  dri: "noahtalerman"
+-
+  task: "Quality check"
+  startedOn: "2024-03-01" 
+  frequency: "Daily" 
+  description: "Every day, Head of Product design will review the “Settled” column on the drafting board to ensure all product action items are complete." 
+  moreInfoUrl: 
+  dri: "noahtalerman"
+-
```

Comment:
```suggestion
```

We can retire this one. Expectation is that items won't move to "Settled" until are product items are complete.